packages/nimble-components/src/combobox/specs/IxD.md

```diff
@@ -0,0 +1,212 @@
+# Nimble Combobox (IxD)
+
+## Overview
+The combobox component is used to present a pre-defined list of options that end-users can choose from or type to create a custom value. Comboboxes combine the flexibility of text input with the efficiency of selection for more user-friendly interactions.
+
+> THE VISUAL DESIGN IN THIS DOCUMENT MAY NOT BE ACCURATE
+
+### Background
+- [Select and Combobox Refresh HLD (#1303)](https://github.com/ni/nimble/issues/1303)
+- [ViD Combobox Styles (Figma)](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1529-41777&mode=design)
+
+## Usage
+When to use:
+- To present a pre-defined list of options
+- To allow end-users to add custom values and/or options
+- For entering settings or preference values that don't (yet) exist
+- For single selection
+
+When not to use:
+- When the options are well-known and straightforward
+- When there are fewer than three options
+- For complex and/or hierarchical data
+- For multi selection
```

Comment:
I'm not sure I know what "When the options are well-known and straightforward" means. The guidance I would give on when to use a `Combobox` is "when the user wants to provide an input for text entry coupled with a popup that allows for quick selection and filtering". The key part of using the Combobox is that it is semantically the same as a `TextField` (and not a `Select`), but comes with "hints" for allowing the user to make a quick selection.

Reply:
I think of a `combobox` as a `text field` and `select` combined. Without a popup menu, the `combobox` might as well be a `text field` with autocomplete and get rid of the dropdown button altogether. But a `combobox` with or without autocomplete, should have a popup menu that indicates selection the same way it does in a `select` component. But anyway, I updated this.